contracts/ToucanRegenBridge.sol

```diff
@@ -90,14 +85,20 @@ contract ToucanRegenBridge is Ownable, Pausable, AccessControl {
     //      Constructor
     // ----------------------------------------
 
-    constructor(address tokenIssuer_, INCTPool nctPool_) Ownable() {
-        tokenIssuer = tokenIssuer_;
+    constructor(
+        INCTPool nctPool_,
+        bytes32[] memory roles,
+        address[] memory accounts
+    ) {
+        require(nctPool != 0xdead, "nctPool must be set");
```

Comment:
Also can you add a unit test to cover this error?

Reply:
Definitely, but I gotta ask, why is `0xdead` a no-no?

Reply:
Apparently, passing a `0` address would fail anyway with `Error: invalid address (argument="address", value="0x0000000000000000000000000000000000000000000000000000000000000000", code=INVALID_ARGUMENT, version=address/5.6.0)` so we may not need to add the `require`. Let me double-check.